### PR TITLE
Update Listening to the protocol documentation with CDP editor

### PR DESCRIPTION
**Pull request by hadrijau**

### CODE REVIEW

File: pages/index.md [hunk 1]
```diff
@@ -89,6 +89,7 @@ instrumenting them. See the <a href="#endpoints">HTTP Endpoints</a> section belo
 <p>This is especially handy to understand how the DevTools frontend makes use of the protocol.
 You can view all requests/responses and methods as they happen.
 
+<p>You can also open a CDP editor to by clicking on the icon on the left of the input bar :
```
Comment (OrKoN) [inline pages/index.md:92]: did you mean to add this line?

File: pages/index.md [hunk 2]
```diff
@@ -89,6 +89,7 @@ instrumenting them. See the <a href="#endpoints">HTTP Endpoints</a> section belo
 <p>This is especially handy to understand how the DevTools frontend makes use of the protocol.
 You can view all requests/responses and methods as they happen.
 
+<p>You can also open a CDP editor to by clicking on the icon on the left of the input bar : 
 <figure class="screenshot">
   <a href="images/protocol-monitor.png" target="_blank" style="text-align: center; display:block;">
```
Comment (OrKoN) [inline pages/index.md:94]: let's update the existing screenshot?

File: pages/index.md [hunk 3]
```diff
@@ -104,6 +105,23 @@ type the command into the prompt at the bottom of the Protocol Monitor panel and
...
+
+<figure class="screenshot">
+  <a href="images/open-editor-screenshot.png" target="_blank" style="text-align: center; display:block;">
+    <img src="images/open-editor-screenshot.png" height="300" alt="Screenshot of the Protocol Monitor" loading="lazy">
+  </a>
+</figure>
+
+<p>This will automatically open the editor which will enable you to better edit your commands. First enter the command you want to send. The corresponding parameters will then appear and you will just have to edit them and either press CTRL + Enter or click the send icon to send your command.
+
+<figure class="screenshot">
```
Comment (OrKoN) [inline pages/index.md:118]: let's just keep one screenshot with the editor open and remove the one with the editor closed

File: pages/images/open-editor-screenshot.png
Comment (OrKoN) [inline pages/images/open-editor-screenshot.png:1]: delete this one?

File: pages/index.md [hunk 4]
```diff
@@ -104,6 +104,15 @@ type the command into the prompt at the bottom of the Protocol Monitor panel and
 <code>Page.captureScreenshot</code>. If the command requires parameters, provide them as JSON, for example,
 <code>{"command":"Page.captureScreenshot","parameters":{"format": "jpeg"}}</code>.
 
+<p> You can also open a CDP editor by clicking on the icon on the left of the input bar. This will automatically open an editor which will enable you to better edit your commands. First, enter the command you want to send. The corresponding parameters will then appear and you will just have to edit them and either press CTRL + Enter or click the send icon to send your command.
```
Comment (OrKoN) [inline pages/index.md:107]:
```suggestion
<p> By clicking on the icon next to the command input (available starting from Chrome version 117.0.5936.0), you can open the command editor. After you select a CDP command, the editor creates a structured form based on the protocol definitions that allows you to edit the parameters, and see the documentation and types for specific parameters. Send the commands by clicking on the send button or using <code>Ctrl + Enter</code>.  Use the context menu in the list of previously sent commands to open one of them in the editor.
```

**Review by OrKoN — APPROVED** [review]

**Comment by mathiasbynens** [comment]
LGTM but could you please losslessly compress the images through [ImageOptim.app](https://imageoptim.com/mac) (if you’re on macOS) or [Squoosh](https://squoosh.app/)?

File: pages/index.md [hunk 5]
```diff
@@ -104,6 +104,15 @@ type the command into the prompt at the bottom of the Protocol Monitor panel and
 <code>Page.captureScreenshot</code>. If the command requires parameters, provide them as JSON, for example,
 <code>{"command":"Page.captureScreenshot","parameters":{"format": "jpeg"}}</code>.
 
+<p> By clicking on the icon next to the command input (available starting from Chrome version 117.0.5936.0), you can open the command editor. After you select a CDP command, the editor creates a structured form based on the protocol definitions that allows you to edit the parameters, and see the documentation and types for specific parameters. Send the commands by clicking on the send button or using <code>Ctrl + Enter</code>.  Use the context menu in the list of previously sent commands to open one of them in the editor.
```
Comment (OrKoN) [inline pages/index.md:107]:
```suggestion
<p> By clicking on the icon next to the command input (available starting from Chrome version 117.0.5936.0), you can open the command editor. After you select a CDP command, the editor creates a structured form based on the protocol definitions that allows you to edit parameters, and view their documentation and types. Send the commands by clicking on the send button or using <code>Ctrl + Enter</code>.  Use the context menu in the list of previously sent commands to open one of them in the editor.
```

Comment (OrKoN) [reply pages/index.md:107]: the previous suggestion was outdated

**Review by mathiasbynens — APPROVED** [review]

**Pull request merged**